src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,81 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String rezalt = readFile(fromFileName);
+        writeFile(toFileName, rezalt);
+    }
+
+    private String readFile(String fromFileName) {
```

Comment:
It's better to split this method into 2 smaller methods.
One of them will read from file and second will create report.
See my comment in "main" method